python/dgllife/data/pdbbind.py

```diff
@@ -151,8 +154,23 @@ def _read_data_files(self, pdb_version, subset, load_binding_pocket):
             self.df = pd.DataFrame(contents, columns=(
                 'PDB_code', 'resolution', 'release_year',
                 '-logKd/Ki', 'Kd/Ki', 'cluster_ID'))
-
+        
+        ##remove core set from refine set
         pdbs = self.df['PDB_code'].tolist()
+        if remove_coreset_from_refinedset:
+            if pdb_version == 'v2015':
+                with open(extracted_data_path + '/v2015/INDEX_core_data.2013','r') as f:
+                    for line in f:
+                        line = line.strip().split()
+                        if line[0] != "#" and line[0] in pdbs:
+                            pdbs.remove(line[0])
```

Comment:
use more accurate name.
```suggestion
                        fields = line.strip().split()
                        if line[0] != "#" and fields[0] in pdbs:
                            pdbs.remove(fields[0])
```